test/sagemaker_tests/pytorch/training/integration/local/test_single_machine_training.py

```diff
@@ -37,10 +37,8 @@ def test_mnist(docker_image, processor, instance_type, sagemaker_local_session,
 
 @pytest.mark.integration("fastai")
 @pytest.mark.model("mnist")
+@pytest.mark.skip_py2_containers
```

Comment:
I think you will have to use

https://github.com/aws/deep-learning-containers/blob/master/test/sagemaker_tests/pytorch/inference/conftest.py#L198-L201

Reply:
That's for inference right?
I'm doing for PT SM Training integration local Test 
https://github.com/aws/deep-learning-containers/blob/6c743b1a36d2ab48d13a4472b3fd9e6e4c228aa3/test/sagemaker_tests/pytorch/training/conftest.py#L245-L249

Reply:
Ah, sure. My bad. Will resolve this.